src/styles/app/21-ai-composer-context.css

```
@@ -142,37 +72,153 @@
-	overflow: hidden;
-	padding: 0 0 4px;
-	border: 1px solid color-mix(in srgb, var(--border-default) 82%, transparent);
-	background: var(--bg-tertiary);
-	box-shadow: 0 0 0 0.5px rgba(0, 0, 0, 0.08), 0 1px 3px rgba(0, 0, 0, 0.06);
-	transition: border-color var(--transition-base), box-shadow
-		var(--transition-base);
+	border: 1px solid color-mix(in srgb, var(--border-default) 35%, transparent);
+	transition: border-color var(--transition-fast), background
+		var(--transition-fast);
+	white-space: pre-wrap;
+	word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)